src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+
+    private String greet;
+    private String greeting;
```

Comment:
```suggestion
    private String greeting;
```

red - old state (current state) of the code
green - suggested changes

here I ask you to remove `greet` variable
you need to replace it with `greeting` where it occurs